scene/main/viewport.cpp

```diff
@@ -1933,19 +1933,18 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
 
 				// If the tooltip timer isn't running, start it.
```

Comment:
Please fix these comments to match the new behavior. We only restart the timer whenever the control changes or the mouse moves too much.

Reply:
Good catch. Let's keep the comments up to date.